lib/services/rocket.py

```diff
@@ -134,6 +137,18 @@ def get_rocketpy_nose(nose: NoseCone) -> RocketPyNoseCone:
         rocketpy_nose.position = nose.position
         return rocketpy_nose
 
+    @classmethod
+    def get_rocketpy_finset_list_from_fins_list(
+        cls, fins_list: List[Fins]
+    ) -> List[RocketPyFins]:
+        rocketpy_finset_list = []
+        if not fins_list:
+            return []
+        for fins in fins_list:
+            rocketpy_fins = cls.get_rocketpy_finset(fins, fins.fins_kind)
+            rocketpy_finset_list.append(rocketpy_fins)
+        return rocketpy_finset_list
```

Comment:
noblock: you could use list comprehension here to become more readable.

Reply:
Thanks for the review!

I could now think in a way to use list comprehension here, could you please share what you thought?

Reply:
Any clue @coderabbitai ?

Reply:
and what about `rocketpy_finset_list.append(rocketpy_fins)` ?

Reply:
and what about rocketpy_finset_list.append(rocketpy_fins) https://github.com/coderabbitai ?

Reply:
and what about `rocketpy_finset_list.append(rocketpy_fins)` @coderabbitai ?

Reply:
Right in front of me 🤦🏻 , thanks!

Reply:
Can you suggest that commit, please? @coderabbitai

Reply:
I mean, can you commit it please?

Reply:
patched in [dc0138f](https://github.com/RocketPy-Team/Infinity-API/pull/33/commits/dc0138f7a7274eb498e0db8823d75be3cde2ce6d)